use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+# Table of Contents
+
+- [Introduction](#introduction)
+  * [Helpful Links before starting](#helpful-links-before-starting)
+  * [Steps to enable Intel QAT in Kata Containers](#steps-to-enable-intel-qat-in-kata-containers)
+  * [Script variables](#script-variables)
+    + [Set environment variables (Every Reboot)](#set-environment-variables--every-reboot-)
+  * [Prepare the Host (Clear Linux)](#prepare-the-host--clear-linux-)
+    + [Identify which PCI Bus the Intel QAT card is on](#identify-which-pci-bus-the-intel-qat-card-is-on)
+    + [Install necessary bundles for Clear Linux and disable kernel updates](#install-necessary-bundles-for-clear-linux-and-disable-kernel-updates)
+    + [Replace Clear Linux kernel with LTS kernel](#replace-clear-linux-kernel-with-lts-kernel)
+    + [Allow insecure modules to load to host](#allow-insecure-modules-to-load-to-host)
+    + [Download Intel QAT drivers](#download-intel-qat-drivers)
+    + [Compile Intel QAT drivers against host kernel](#compile-intel-qat-drivers-against-host-kernel)
+    + [Copy Intel QAT configuration files and enable VF’s](#copy-intel-qat-configuration-files-and-enable-vf-s)
```

Comment:
The link address is wrong here - it should be `copy-intel-qat-configuration-files-and-enable-vfs` according to https://github.com/kata-containers/tests/tree/master/cmd/check-markdown.

Reply:
I ran the kata-check-markdown to generate TOC so the next version should have fixed links. I'll check each link before i resubmit.